submissions/ksalpern/Hooli-style-Popup/index.html

```diff
@@ -0,0 +1,275 @@
+                        class="checkbox__more"
+                        type="checkbox"
+                      />
+                      <label for="checkbox__more">
+                        <span class="popup__more-name">More apps</span>
+                      </label>
+
+                      <ul class="popup__more-items">
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/news.png"
+                              alt="News"
+                              class="icon"
+                            />
+                            <span class="popup__more-name">News</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/photos.png"
+                              alt="Photos"
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Photos</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/promo.png"
+                              alt="Promo"
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Promo</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/search.png"
+                              alt="Search"
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Search</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/tanslation.png"
+                              alt="Translation"
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Translate</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/videos.png"
+                              alt="Videos"
```

Comment:
Fix styles on `:hover`: avoid overlapping text by the background of the hovered menu item
![image](https://user-images.githubusercontent.com/16196199/182661527-91be2f26-3664-467c-a7a2-ca9fe628544c.png)